backend/gn_module_monitoring/command/cmd.py

```diff
@@ -183,6 +184,29 @@ def cmd_install_monitoring_module(module_code):
     return
 
 
+@click.command("clean_existing_permissions")
+@click.argument("module_code", required=False, default="")
+@with_appcontext
+def cmd_clean_existing_permissions(module_code):
+    """
+    Nettoyage des permissions disponibles
+        Mise à jour des objets permissions associés au module
+        Transfert des droits pour les utilisateurs
+        Nettoyages des permissions et des permissions disponibles
+
+    """
+    if module_code:
+        process_available_permissions(module_code, session=DB.session)
+        clean_existing_permissions(module_code, session=DB.session)
+        DB.session.commit()
+        return
+
+    for module in installed_modules():
+        process_available_permissions(module["module_code"], session=DB.session)
+        clean_existing_permissions(module["module_code"], session=DB.session)
+    DB.session.commit()
```

Comment:
**Suggestion:** Consider using a context manager for handling database sessions to ensure that resources are properly cleaned up. [best practice]
```suggestion
    if module_code:
        with DB.session.begin():
            process_available_permissions(module_code, session=DB.session)
            clean_existing_permissions(module_code, session=DB.session)
        return

    for module in installed_modules():
        with DB.session.begin():
            process_available_permissions(module["module_code"], session=DB.session)
            clean_existing_permissions(module["module_code"], session=DB.session)
```